src/config.c

```diff
@@ -2539,6 +2539,12 @@ int updateClusterFlags(const char **err) {
     return 1;
 }
 
+static int updateClusterPort(const char **err) {
```

Comment:
```suggestion
static int updateClusterAnnouncedPorts(const char **err) {
```

The cluster port is a specific thing.